README.md

```diff
@@ -37,13 +37,32 @@ through ADB, so you can use your Android device as a USB microphone.
 ## Usage
 
 1. Enable *Android Debug Bridge* (ADB) from the *Developer options* and connect
-   the device to your computer.
+   the device(s) to your computer.
+
 2. Run `./audiosource run` to start Audio Source and forward the audio
-   automatically. (You may have to grant the permission to record audio in
-   Android.)
-3. Run `./audiosource volume LEVEL`, to set the PulseAudio source volume to
-   LEVEL, for instance `200%` (you will likely need to set the volume higher
-   than 100%).
+   automatically. You may have to grant the permission to record audio in
+   Android.
+
+   **NOTE:** If you have multiple devices connected then you will have to
+   specify the serial number of the device you would like to forward audio to
+   with `./audiosource run -s SERIAL` (device serial numbers can be found by
+   running `adb devices`).
+
+   Additionally, you may override the default PulseAudio source name with
+   `./audiosource run -n NAME`. Otherwise, a default name of
+   `audiosource<RANDOM_UUID>` will be used.
+
+   You can also utilize job control to connect multiple devices to PulseAudio
+   simultaneously.
+
+   ```
+   $ ./audiosource run -s SERIAL_ONE 1> /dev/null &
+   $ ./audiosource run -s SERIAL_TWO 1> /dev/null &
```

Comment:
```suggestion
   $ ANDROID_SERIAL=serial1 ./audiosource run 1> /dev/null &
   $ ANDROID_SERIAL=serial2 ./audiosource run 1> /dev/null &
```

Here you capture stdout, but the name of source should be printed in the forwarding message.